README.md

```diff
@@ -1510,6 +1510,7 @@ A registry allows you to publish your Rust libraries as crate packages, to share
   * [stdx](https://github.com/brson/stdx) — Learn these crates first as an extension to std
   * [University of Pennsylvania's Comp Sci Rust Programming Course](http://cis198-2016s.github.io/schedule/)
   * [Build a language VM](https://blog.subnetzero.io/post/building-language-vm-part-00/)
+  * [Code Playground](https://codeplayground.app) - Interactively edit & play rust snippets on your mobile device.
```

Comment:
Can you please clarify that's iPhones and iPads only please? This would be useful for me occasionally, but all my "mobile devices" are Android.

Reply:
@palfrey Sure. Just pushed a commit with the change.